researchapp/services/logging.py

```diff
@@ -0,0 +1,42 @@
+""" Log stuff to ElasticSearch so we can look at it later. """
```

Comment:
Is this something that should eventually be shared code between research app and test app?